RetoEuskalmet/src/main/java/controlador/controladorInsertarActualizar.java

```diff
@@ -0,0 +1,77 @@
+package controlador;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import org.json.JSONException;
+
+import modelo.DescargarJSON;
+import modelo.JSONtoXML;
+import modelo.InsertarDatosGeograficos;
+import modelo.DescargarDatosMetereologicos;
+import modelo.InsertarDatosAtmosfericos;
+import modelo.InsertarRelacionExiste;
+
+public class controladorInsertarActualizar {
+
+	public String ejecutar() throws FileNotFoundException, JSONException, IOException {
+		boolean existen = false;
+		String texto = "";
+		
+		File archivoMunicipios = new File("./archivos/municipios.json");
+		File archivoIndex = new File("./archivos/index.json");
+		File archivoEstaciones = new File("./archivos/estaciones.json");
+		File archivoEspNaturales = new File("./archivos/espacios-naturales.json");
+		
+		if (archivoIndex.exists()) {
+		    existen = true;
+		}
+		if (archivoMunicipios.exists()) {
+		    existen = true;
+		}
+		if (archivoEstaciones.exists()) {
+		    existen = true;
+		}
+		if (archivoEspNaturales.exists()) {
+		    existen = true;
+		}
+	
+		if (!existen) {
+		    DescargarJSON descargarJSON = new DescargarJSON();
+		    JSONtoXML jsonToXml = new JSONtoXML();
+		    InsertarDatosGeograficos insertarDatosGeograficos = new InsertarDatosGeograficos();
+		    InsertarRelacionExiste insertarRelacionExiste = new InsertarRelacionExiste();
+		    DescargarDatosMetereologicos descargarDatosMetereologicos = new DescargarDatosMetereologicos();
+		    InsertarDatosAtmosfericos insertarDatosAtmosfericos = new InsertarDatosAtmosfericos();
+		    
+	
+		    descargarJSON.DescargarJSON();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.